quiche/quic/core/quic_flow_controller_test.cc

```diff
@@ -200,7 +200,7 @@ TEST_F(QuicFlowControllerTest, ReceivingBytesFastIncreasesFlowWindow) {
   EXPECT_CALL(*session_, WriteControlFrame(_, _)).Times(1);
   EXPECT_TRUE(flow_controller_->auto_tune_receive_window());
 
-  // Make sure clock is inititialized.
+  // Make sure clock is initialized.
```

Comment:
this repository appears to be very heavily en-GB. That should make this `initialised` (and similar biases in a couple of other corrections), it was too much of a spelling shift for me to apply, so I've made the simplest correction plus this note.